go/tools/releaser/upgradedep_test.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	bzl "github.com/bazelbuild/buildtools/build"
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+)
+
+const (
+	reposFile = "go/tools/releaser/testdata/repositories.bzl"
+	funcName  = "go_rules_dependencies"
```

Comment:
```suggestion
	_reposFile = "go/tools/releaser/testdata/repositories.bzl"
	_funcName  = "go_rules_dependencies"
```

[Prefix Unexported Globals with _](https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_)